### PR TITLE
feat: Add `language/keyword-extraction`

### PR DESCRIPTION
**Pull request by kysely**
## Description

Adding _“Keyword Extraction”_ ML capability, so far only with `ibm-cloud-natural-language-understanding` provider for now.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: grid/language/keyword-extraction/maps/__snapshots__/mock.test.ts.snap [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`language/keyword-extraction/mock ExtractKeywords extracts all keywords from the phrase 1`] = `
```
Comment (Jakub-Vacek) [inline grid/language/keyword-extraction/maps/__snapshots__/mock.test.ts.snap:3]: nipick: using test helper with map for mock provider can be kinda confusing :/

File: grid/language/keyword-extraction/profile.supr [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  Extracted Keyword
+  The keyword or phrase extracted from the given text.
+  """
+  keyword! string!
+
+  """
+  Importance of Keyword
+  A number between 0 and 1 indicating importance of the keyword within the given text.
+  """
+  importance number
```
Comment (Jakub-Vacek) [inline grid/language/keyword-extraction/profile.supr:88]: Question: `importance` is nullable and optional to be able to harmonize (future) providers?
Maybe we could mention optionality in description.

**Review by Jakub-Vacek — APPROVED** [review]
I don't see anything wrong 🙂 Nice work.👍

**Comment by kysely** [comment]
@Jakub-Vacek Can you please check again? I used a better nested result. Instead of returning the array directly, it returns an object with `keywords` inside

**Review by Jakub-Vacek — APPROVED** [review]
👍

**Pull request merged**